### DISCUSSION TITLE
New column in the "Authorization Page"

### DISCUSSION THREAD

**Discussion by sylwester-majcher** [lithnet/access-manager#76] [Ideas]
Hi All,
My proposal is to add new column, the name will be "Access Type" and the value will come from "type of acces" that we have giveen. For now from the first screen of "Authorization Page" we cannot see our rule comes from JIT, LAPS or Bitlocker type, etc.
Do you think is it possible?
Best regards
Sylwester

**Comment by ryannewington** [comment]
I think this will be a good idea. I've moved this issue to a [new enhancement item](https://github.com/lithnet/access-manager/issues/81) for tracking
